src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(person => person.sex === 'm');
+
+  men = century ? men.filter(
+    person => Math.ceil(person.died / 100) === century
+  ) : men;
```

Comment:
```suggestion
  const men = people.filter(person =>
    century
      ? Math.ceil(person.died / 100) === century && person.sex === 'm'
      : (person.sex === 'm')
```
Можна зразу ж все фільтрувати набагато коротше (такий підхід також можна застосувати і для двох наступних функцій)